kubernetes/start

```diff
@@ -29,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup
+./cleanup || true
```

Comment:
if you don't want it to fail or complain and just pass silently if there's nothing to remove, you can also do
```
for i in `docker ps -a | grep -iv name | awk '{print $1}'`; do docker rm -vf $i; done
```
just mentioning because it's my favorite :)

with docker 1.13 it's nicer
```
vpalade:codeflavor$ docker system prune
WARNING! This will remove:
	- all stopped containers
	- all volumes not used by at least one container
	- all networks not used by at least one container
	- all dangling images
Are you sure you want to continue? [y/N] y
Deleted Volumes:
6f9ed8fcce60ad75f525baa42a0c3444d9be81294357cc37f37b5b89f947c644
b2788412fb1df308f60e7e877abeb418ea03c0d47437b36facb78e374c766402
bd3920f73bb15e7f05636248bf54c12271fa582dc7dfd6464020c682223f2437
e632d0d82b0fda0f30a204ce55b43f719661a7dd2f5aa81558052a198be25e23
203af67358fab87e30b2df15a50ea06f191fea2203d810349a55b5bc38851c33
d1948ab203a8af263d033f6acb4ddc22a00077d98264f21e28c81efea58e8f60
df5ce42dd760183573ae928b2c0e3c054fa0131b0526b9447dc702cb344d711c
e9c731350126800da8554f58a93c5c58ae4309deb04ccb17ecbda526cc7f3f73
ccdbaf01eaf6b16abf095adbeceba0174d02ca6b78a9359eaf8f5dfad92a07ac

Total reclaimed space: 1.904 GB
```